src/sentry/auth_v2/serializers.py

```diff
@@ -0,0 +1,47 @@
+# import serializer from rest_framework
+
+from datetime import datetime
+from typing import TypedDict
+
+from rest_framework.request import Request
+
+from sentry.api.serializers import Serializer
+
+from .types import SessionData
+
+
+class SessionSerializerResponse(TypedDict, total=False):
```

Comment:
seems like the total=False should't be there?

Reply:
I'll remove it in a subsequent PR.